src/components/PlioQuestion.vue

```diff
@@ -131,28 +149,65 @@ export default {
     
     // Highlights the correct option as green, wrong one as red
     showResult(){
-      if (!this.isAnswerCorrect) 
-        this.$refs[this.selectedOption].className = "answer_option-wrongAnswer"
+      if (!this.isAnswerCorrect) {
+        this.$refs[this.selectedOption].classList.add("wrongAnswer")
+      }
+      
+      this.$refs[this.correctAnswer].classList.add("correctAnswer")
+    },
```

Comment:
Broke down class
 `answer_option-wrongAnswer` into two classes => `answer_option` and `wrongAnswer`
and
 `answer_option-correctAnswer` into two classes => `answer_option` and `correctAnswer`